README.md

```diff
@@ -280,6 +280,11 @@ You can alternatively develop the app via [Docker][], which
 means you don't have to install any dependencies. However,
 Docker takes a bit of time to learn how to use.
 
+After you install Docker on your machine, open up `Preferences > Resources > Advanced`
+and make sure you give it at least 4 GB of memory to play with.
+If you don't, you might get an out-of-memory error when attempting
+to build the Docker image.
```

Comment:
Nice!  Maybe change this to `build and/or run the Docker image`?  I think that the funky error you were experiencing during development (if this setting is related to that) was actually happening during the building of the JS bundles, which was part of the _running_ of the Docker image, rather than the building of it.  But for production deploys I think it might be part of the _building_ of the Docker image, since the JS bundles are "baked into" the image itself in that situation, so an "and/or" seems appropriate here.  Oy it's complicated.

Reply:
Very good point! Thank you.